src/examples/get_available_disbursement_banks.php

```diff
@@ -0,0 +1,12 @@
+<?php
+    require('config/xendit_php_client_config.php');
+    require('XenditPHPClient.php');
+    
+    $options['secret_api_key'] = constant('SECRET_API_KEY');
+    $options['server_domain'] = constant('SERVER_DOMAIN');
```

Comment:
We should not need to set the server domain. Server domain should already be part of the library. I thought we implemented this a long time ago.